parquet/src/main/java/com/netflix/iceberg/parquet/ParquetMetricsRowGroupFilter.java

```diff
@@ -156,6 +157,13 @@ public Boolean or(Boolean leftResult, Boolean rightResult) {
       Preconditions.checkNotNull(struct.field(id),
           "Cannot filter by nested column: %s", schema.findField(id));
 
+      // When filtering nested types notNull() is implicit filter passed even
+      //   though complex filters aren't pushed down in Parquet. Leave all
+      //    nested column type filters to be evaluated post scan.
```

Comment:
Style: comments should be wrapped at column 100 and should not contain the beginning whitespace.